internal/fwschemadata/data_get_at_path_test.go

```diff
@@ -290,7 +291,8 @@ func TestDataGetAtPath(t *testing.T) {
 					path.Root("bool"),
 					"Value Conversion Error",
 					"An unexpected error was encountered trying to build a value. This is always an error in the provider. Please report the following to the provider developer:\n\n"+
-						"unhandled unknown value",
+						"Received unknown value, however the target type cannot handle unknown values. Use the corresponding `types` package type or a custom type that handles unknown values.\n\n"+
+						"Path: bool\nTarget Type: *bool\nSuggested Type: types.BoolType",
```

Comment:
Ah, okay. The suggested type here should be `types.Bool` for now, but stripping the `Type` suffix would only temporarily work for the framework `types` types and isn't a long term solution. The `attr.Type` interface doesn't give us a way to really know the correct/suggested `attr.Value` type and maybe it should. Technically, we could use the `attr.Value` returned from the `ValueFromTerraform` method if called with a null value of the `TerraformType`, but custom types don't _have_ to support null values if they're always required 🙁 . I'll create an issue to consider adding something to that interface so we can recommend things correctly.

For now though, I'm not sure if there's okay enough messaging we can give here to say "use the value type associated with the type". We might be able to merge this in as-is right now and try to fix that up before the next release.

Reply:
We could alter the wording in the diagnostic until [attr: Consider Adding ValueType Method to Type Interface](https://github.com/hashicorp/terraform-plugin-framework/issues/496#top) is addressed.

Perhaps something like:
```
Path: path.String()
Target Type: target.Type()
Suggested `types` Type: typ.String() (use the value type associated with the type e.g., use types.Object for types.ObjectType)
Suggested Pointer Type: *target.Type()
```